src/adaptix/_internal/morphing/load_error.py

```diff
@@ -130,6 +130,14 @@ class BadVariantError(LoadError):
     input_value: Any
 
 
+@custom_exception
+@dataclass(eq=False)
+class MultipleBadVariantError(LoadError):
```

Comment:
New exception naming policy suppose that name signaling error shouldn't have ``Error`` suffix.
Old exception will be renamed. It's ok that name of new exception is inconsistent with old one.